### DISCUSSION TITLE
if react_component("HelloWorld") fails, can we have it raise or warn?

### DISCUSSION THREAD

**Discussion by jasonfb** [shakacode/react_on_rails#1487] [Ideas]
if `react_component("HelloWorld")` fails, can we have it raise or warn?
perhaps optionally raise or want based on configuration

**Comment by Judahmeek** [comment]
Does the [raise_on_prerender_error](https://github.com/shakacode/react_on_rails/blob/master/docs/api/view-helpers-api.md#react_component) option solve your issue?
